src/test/config.sol

```diff
@@ -1633,6 +1633,36 @@ contract Config {
             calc_cut:     9990,
             lerp:         false
         });
+        afterSpell.collaterals["DIRECT-COMPV2-DAI"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      5 * MILLION,
+            aL_gap:       5 * MILLION,
+            aL_ttl:       12 hours,
+            line:         0,
+            dust:         0,
+            pct:          0,
+            mat:          10000,
+            liqType:      "",
+            liqOn:        false,
+            chop:         0,
+            cat_dunk:     0,
+            flip_beg:     0,
+            flip_ttl:     0,
+            flip_tau:     0,
```

Comment:
maybe a `d3m_tau` could be included with the respective range check (same could be done for `plan.barb` I suppose)

Reply:
These are generic `ilk`s values. IMO we shouldn't be adding specific things. That should leave in some other place. Where is a good question...

Reply:
an ad-hoc d3m config for monitoring was already suggested